test/Microsoft.TestPlatform.CrossPlatEngine.UnitTests/TestableImplementations/TestableTestRunCache.cs

```diff
@@ -9,7 +9,7 @@ namespace TestPlatform.CrossPlatEngine.UnitTests.TestableImplementations
     using Microsoft.VisualStudio.TestPlatform.ObjectModel;
     using Microsoft.VisualStudio.TestPlatform.ObjectModel.Client;
 
-    public class TestableTestRunCache : ITestRunCache
+    internal class TestableTestRunCache : ITestRunCache
```

Comment:
I assume this change won't hurt us? Since we're closing down a publicly available class?

Reply:
It should be fine. It's a class in test project, and I had to reduce it's visibility because the TestResultMetric class would otherwise be less visible.